handlers/group.ts

```diff
@@ -0,0 +1,62 @@
+import {randomUUID} from 'expo-crypto';
+interface IGroup {
+    create(groupName: string, action : () => void): void;
+    join(uuid: string, action : (ack: Boolean) => void): void;
+    checkIfInGroup(action : (ack: Boolean) => void): void ;
+    leave() : Boolean;
+    setPeers() : void;
+}
+
+class GroupHandle implements IGroup {
+
+    constructor(private gun: Gun) {        
+    }
+
+    public create(groupName: string, action : () => void): void {
+        let user = gun.user(userPub)
+        let groupId : string = randomUUID();        
+        let context = gun.get("groups").set({groupId: groupId})
+        context.get("members").set({members: userPub})
+        context.get("name").set({groupName: groupName})        
+        user.get("group").put({groupId: groupId})
+        action()
+    }
+
+    public join(uuid: string, action : (ack: Boolean) => void): void {           
+        gun.get("groups").once(
+            function(data, key) {
+                if (data != uuid){
+                    user = gun.user(userPub)
+                    gun.get("groups").get(data).set({members: userPub})
```

Comment:
I think it should be `gun.get("groups").get(key).set({members: userPub})` or `gun.get("groups").get(key).get("members").set(userPub)` not 100% though